v5/v5_stable.ino

```diff
@@ -1547,51 +1518,38 @@ void inputManager(String command, int inputFrom) {
       setTime(tempTime);
     }
     println("Time String : " + rtc);
-    // println(Time.Date);
-    // println(Time.hour);
-    // println(Time.minutes);
-  }
-  // else if (command.indexOf("!") != -1) {
-  //   if (command.indexOf("r11") != -1) {
-  //     digitalWrite(BATTERY_PAIR_SELECTOR, LOW);
-  //     delay(1000);
-  //   } else if (command.indexOf("r10") != -1) {
-  //     digitalWrite(BATTERY_PAIR_SELECTOR, HIGH);
-  //     delay(1000);
-  //   } else if (command.indexOf("r2a1") != -1) {
-  //     digitalWrite(BATTERY_CHARGER, LOW);
-  //     delay(1000);
-  //   } else if (command.indexOf("r2a0") != -1) {
-  //     digitalWrite(BATTERY_CHARGER, HIGH);
-  //     delay(1000);
-  //   } else if (command.indexOf("r2b1") != -1) {
-  //     digitalWrite(POWER_OUTPUT_SELECTOR, LOW);
-  //     delay(1000);
-  //   } else if (command.indexOf("r2b0") != -1) {
-  //     digitalWrite(POWER_OUTPUT_SELECTOR, HIGH);
-  //     delay(1000);
-  //   }
-  // }
-  // else if (command.indexOf("switch") != -1) {
-  //   if (command.indexOf("0") != -1)
-  //     digitalWrite(BATTERY_PAIR_SELECTOR, HIGH);
-  //   else if (command.indexOf("1") != -1)
-  //     digitalWrite(BATTERY_PAIR_SELECTOR, LOW);
-  //   else
-  //     println("Error in switch command");
-  // } else if (command.indexOf("chargeFor") != -1) {
-  //   batteryChargeTime = fetchNumber(getCompleteString(command, "chargeFor"));
-  //   println("Battery charge time updated to : " + String(batteryChargeTime) +
-  //           " minutes");
-  //   inputFrom == 3 ? command += "<executed>" : "";
-  // }
-  else if (command.indexOf("updateTime") != -1) {
+  } else if (command.indexOf("updateTime") != -1) {
     println("Updating time");
     sendSMS("#setTime", "+923374888420");
     inputFrom == 3 ? command += "<executed>" : "";
   } else if (command.indexOf("whatsapp") != -1) {
-    sendWhatsappMsg("test_message_from_esp32");
+    if (command.length() <= 8)
+      sendWhatsappMsg("test_message_from_esp32");
+    else
+      sendWhatsappMsg(command.substring(8, -1));
     inputFrom == 3 ? command += "<executed>" : "";
+  } else if (command.indexOf("readSPIFFS") != -1) {
+    println("Data in SPIFFS : " + readSPIFFS());
+  } else if (command.indexOf("value of:") != -1) {
+    String varName =
```

Comment:
- You have read and update variables but you are not using those yet, use at least 2 SPIFFS variables.
- this part is defined as a function I'll suggest to use it to reduce complexity